webapp/src/dogsList/listWithDogs.tsx

```diff
@@ -1,6 +1,6 @@
 import "date-fns";
 import React from "react";
-import { Button } from "@material-ui/core";
+import { Button, Card, CardHeader, List } from "@material-ui/core";
```

Comment:
CardHeader is not used I guess. As well as set and remove cookie. So you can remove them